README.md

```diff
@@ -263,15 +291,17 @@ The API Supports a number of options to configure the read
   <tr valign="top">
    <td><code>dataset</code>
    </td>
-   <td>The dataset containing the table.
+   <td>The dataset containing the table. This option is intended to be used only
+   with the <code>load()</code>/<code>save()</code> methods.
        <br/>(Optional unless omitted in <code>table</code>)
    </td>
    <td>Read/Write</td>
   </tr>
   <tr valign="top">
    <td><code>project</code>
    </td>
-   <td>The Google Cloud Project ID of the table.
+   <td>The Google Cloud Project ID of the table. This option is intended to be
```

Comment:
This option should be used with standard table and views, but not when loading query results